janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/rest/RestElasticSearchClient.java

```diff
@@ -76,6 +82,29 @@ public void close() throws IOException {
     }
 
     @Override
+    public int getMajorVersion() {
+        if (majorVersion == null) {
```

Comment:
to un-nest code lets check majorVersion != null first and return, and then do the logic to compute it